text/first-push.md

```diff
@@ -11,7 +11,7 @@ impl List {
 }
 ```
 
-First thing's first, we need to make a node to store our element in:
+First things first, we need to make a node to store our element in:
```

Comment:
[This](http://english.stackexchange.com/questions/7419/which-is-the-correct-idiom-first-things-first-or-first-things-first) gives a good overview.